File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1115.lua
while(true) do
    x = io.read("*number")
    y = io.read("*number")
    if(x==0 or y==0) then
        break
    end
    if(x>0) then
        if(y>0) then
    		print("primeiro")
        else
    		print("quarto")
        end
    else
        if(y>0) then
    		print("segundo")
        else
            print("terceiro")
        end
    end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1115.cpp
#include <iostream>

using namespace std;

int main(){
    int x, y;
    while(true){
        cin >> x >> y;
        if(x==0 or y==0){
            break;
        }
        if(x>0){
            if(y>0){
                cout << "primeiro";
            } else{
                cout << "quarto";
            }
        } else{
            if(y>0){
                cout << "segundo";
            } else{
                cout << "terceiro";
            }
        }
        cout << endl;
    }

    return 0;
}